.changeset/cold-hairs-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'spectacle': patch
+---
+
+security update: increment ts-jest and babel/core
```

Comment:
@carbonrobot Since this was in the root package.json, I thought it would warrant a changeset for the whole package, but since it was just a minor version update I could be convinced to remove this if it avoids an unnecessary release of spectacle.